src/pages/user_profile/ResearcherStatus.js

```diff
@@ -99,15 +99,10 @@ export default function ResearcherStatus(props) {
     ) : (
       <div>
         <p>No Library Card Found</p>
-        <button
-          className='f-left btn-primary common-background'
-          onClick={goToRequestRole}
-          style={{
-            marginTop: '10px',
-            marginBottom: '50px'
-          }} >
-          Request a Library Card
-        </button>
+        <p style={{
+          marginTop: '10px',
+          marginBottom:'50px'
+        }}>You must have a Library Card to submit a data access request. To obtain one, your Institutional Signing Official must register in DUOS, request and receive Signing Official permissions, and issue you a Library Card.</p>
```

Comment:
This leaves the `goToRequestRole` method unused. 
That takes the user to `/request_lc` which now, no longer needs a route in `Routes.js`.
That takes us to `RequestForm` which now no longer uses the prop `isRequestLCPage`
This is essentially the "Request Library Card" feature so this code changes makes that code path unreachable now. 
We should potentially do two things here
1. Confirm with @solideoglori that we no longer have a need for this feature
2. If so, remove all of the unreachable code

Reply:
1. Yes, we no longer need.
2. Agreed!